### DISCUSSION TITLE
How to hide a Input form

### DISCUSSION THREAD

**Discussion by twilliann** [chatwoot/chatwoot#8090] [General]
Hello, I wanted to hide the Inbox Channel field when editing API channels, in ` url/app/accounts/{id}/settings/inboxes/{id)
`
I tried to change the file `/home/chatwoot/chatwoot/app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Api.vue`, but it only changes for creation and not for editing the item.

I have added
`disabled="true"`

like this

```
    <form class="mx-0 flex flex-wrap" @submit.prevent="createChannel()">
      <div class="w-[65%] flex-shrink-0 flex-grow-0 max-w-[65%]">
        <label :class="{ error: $v.channelName.$error }">
          {{ $t('INBOX_MGMT.ADD.API_CHANNEL.CHANNEL_NAME.LABEL') }}
          <input
            v-model.trim="channelName"
            type="text"
            :placeholder="
              $t('INBOX_MGMT.ADD.API_CHANNEL.CHANNEL_NAME.PLACEHOLDER')
            "
            @blur="$v.channelName.$touch"
	    disabled="true"
          />
          <span v-if="$v.channelName.$error" class="message">{{
            $t('INBOX_MGMT.ADD.API_CHANNEL.CHANNEL_NAME.ERROR')
          }}</span>
        </label>
      </div>
```

It works, but it only disables when **creating** a new channel, **not when editing.**
Has anyone tried something similar?

**Comment by sojan-official** [comment]
Thanks for reaching out 🙌  

We don’t currently provide direct support for modifying or customizing the core codebase. If you’re planning to make changes, you would need to maintain those updates on your fork.

If this is something that could benefit others as well, feel free to open a feature request so we can gauge community interest.
